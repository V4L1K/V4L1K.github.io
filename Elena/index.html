<!DOCTYPE html>
<html>
<head>
	<title>ELENA</title>
	<meta charset = "UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="CSS/style.css">
	<link rel="stylesheet" href="CSS/normalize.css">
	<link rel="stylesheet" href="CSS/media.css">
</head>
<body>
<div class="container">
	<div class="header">
		<div class="container_title">
			<div class="logo_img"></div>
			<ul class="container_nav">
				<li><a href="">HOME</a></li>
				<li><a href="">SERVICES</a></li>
				<li><a href="">FEATURES</a></li>
				<li><a href="">CONTACT</a></li>
			</ul>
			<div class="title_h1">HAVE A GOOD DAY</div>
			<div class="container_title_h3">
				<hr class="leftLine"></hr>
				<div class="title_h3">JUST FOR YOU ON THEMEFOREST</div>
				<hr class="rightLine"></hr>
			</div>
			<div class="button">
				<a href="" class="button_text"> GET YOUR COPY NOW</a>
			</div>
		</div>
	</div>
	<div class="container_section">
		<div class="section_1">
			<div class="container_blocks_section_1">
				<div class="block_1">
					<div class="icon icon_block_1"></div>
					<div class="block_h2">PROFESIONALS</div>
					<hr class="block_hr">
					<p class="p_block">
						Lorem ipsum dolor sit amet consectetur elit.
					</p>
				</div>
				<div class="block_2">
					<div class="icon icon_block_2"></div>
					<div class="block_h2">AMATEURS</div>
					<hr class="block_hr">
					<p class="p_block">
						Lorem ipsum dolor sit amet consectetur elit.
					</p>
				</div>
				<div class="block_3">
					<div class="icon icon_block_3"></div>
					<div class="block_h2">FOR EVERYONE</div>
					<hr class="block_hr">
					<p class="p_block">
						Lorem ipsum dolor sit amet consectetur elit.
					</p>
				</div>
			</div>
		</div>
		<div class="section_2">
			<div class="container_article_section_2">
				<div class="article_1">
					<div class="foto_article_1"></div>
					<div class="article_h2">RESPONSIVE EMAIL TEMPLATE</div>
					<div class="article_hr"></div>
					<p class="article_text">
						Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec interdum mollis
					</p>
					<div class="button_read_more"><a href="">Read more</a></div>
				</div>
				<div class="article_2">
					<div class="foto_article_2"></div>
					<div class="article_h2">FULLY COMPATIBLE TEMPLATE</div>
					<div class="article_hr"></div>
					<p class="article_text">
						Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec interdum mollis
					</p>
					<div class="button_read_more"><a href="">Read more</a></div>
				</div>
			</div>
		</div>
		<div class="section_3">
			<div class="container_block_section_3">
				<div class="h2_section_3">
					<div class="icon_section_3"></div>
					FULLY SUPPORTED
				</div>
				<div class="hr_section_3"></div>
				<p class="p_section_3">
					Lorem ipsum dolor sit amet, consectetur adipisicing elit. Tempora, reiciendis. Quos nostrum architecto laborum voluptatum sequi vitae dolores ad aliquam, ratione praesentium quas numquam vel consequuntur alias.
				</p>
				<div class="button_read_more read_more_section_3"><a href="">Read more</a></div>
			</div>
		</div>
		<div class="section_4">
			<div class="container_foto_section_4">
				<div class="article_1_section_4">
					<div class="foto_1_section_4"></div>
					<div class="h2_section_4">FULLY LAYERED PSD</div>
					<div class="hr_section_4"></div>
					<p class="p_section_4">
						Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
					</p>
					<a href="" class="link_section_4">Check this out  ></a>
				</div>
				<div class="article_2_section_4">
					<div class="foto_2_section_4"></div>
					<div class="h2_section_4">FULL COMPATIBILITY</div>
					<div class="hr_section_4"></div>
					<p class="p_section_4">
						Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
					</p>
					<a href="" class="link_section_4">Check this out  ></a>
				</div>
				<div class="article_3_section_4">
					<div class="foto_3_section_4"></div>
					<div class="h2_section_4">VERY GREAT MODULES</div>
					<div class="hr_section_4"></div>
					<p class="p_section_4">
						Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
					</p>
					<a href="" class="link_section_4">Check this out  ></a>
				</div>
			</div>
		</div>
		<div class="section_5">
			<div class="container_articles_section_5">
				<div class="article_1_section_5">
					<div class="h2_section_5">
						<div class="article_1_icon_section_5"></div>
						SEND IT NOW
					</div>
					<div class="hr_section_5"></div>
					<p class="p_section_5">
						Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eveniet quaerat neque, deleniti dicta hic voluptates.
					</p>
					<div class="button_read_more read_more_section_5"><a href="">Read more</a></div>
				</div>
				<div class="article_2_section_5">
					<div class="h2_section_5">
						<div class="article_2_icon_section_5"></div>
						VERY NICE TO SEE
					</div>
					<div class="hr_section_5"></div>
					<p class="p_section_5">
						Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eveniet quaerat neque, deleniti dicta hic voluptates.
					</p>
					<div class="button_read_more read_more_section_5"><a href="">Read more</a></div>
				</div>
			</div>
		</div>
		<div class="section_6">
			<div class="container_foto_section_6">
				<div class="foto_section_6"></div>
				<div class="container_text_section_6">
					<div class="h2_section_6">JUST FOR YOU ON THEMEFOREST</div>
					<div class="hr_section_6"></div>
					<p class="p_section_6">
						Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec lorem ipsum dolor sit amet, consectetur adipisicing.
					</p>
					<div class="button_read_more read_more_section_6" ><a href=""> Read more</a></div>
				</div>
			</div>
		</div>
		<div class="section_7">
			<div class="h2_section_7">
				<div class="icon_section_7"></div>
				THIS TEMPLATE WAS FEATURED ON
			</div>
			<div class="hr_section_7"></div>
			<div class="container_img">
				<div class="img_1_section_7"></div>
				<div class="img_2_section_7"></div>
				<div class="img_3_section_7"></div>
				<div class="img_4_section_7"></div>
			</div>
		</div>
		<div class="footer">
			<div class="copyright"><span> Elenea Â©</span> copyright 2013 - Unsubscribe here</div>
		</div>
	</div>
</div>
<script src = "script.js"></script>
</body>
</html>