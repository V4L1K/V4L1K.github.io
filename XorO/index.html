<!DOCTYPE html>
<html>
<head>
	<title>X or O</title>
	<meta charset = "UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="css/normalize.css">
	<link rel="stylesheet" href="css/media.css">
</head>
<body>
<div id="player">Now do is X</div>
<div id="wrapper">
	<button class="btn" id="btn1"></button>
	<button class="btn" id="btn2"></button>
	<button class="btn" id="btn3"></button>
	<button class="btn" id="btn4"></button>
	<button class="btn" id="btn5"></button>
	<button class="btn" id="btn6"></button>
	<button class="btn" id="btn7"></button>
	<button class="btn" id="btn8"></button>
	<button class="btn" id="btn9"></button>
</div>
<div id="pop_up"></div>
<script src = "script.js"></script>
</body>
</html>
