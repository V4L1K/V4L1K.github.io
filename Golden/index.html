<!DOCTYPE html>
<html>
<header>
	<title>Golden</title>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta name="area.size" content="width=400px,height=300px">
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="css/normalize.css">
	<link rel="stylesheet" href="css/media.css">
</header>
<body>
<div class="header">
	<div class="container-menu">
		<div class="logo"><a href="#">Golden</a></div>
		<ul class="menu">
			<li class="menu__items"><a href="#" id="active">home</a></li>
			<li class="menu__items"><a href="#service">services</a></li>
			<li class="menu__items"><a href="#portfolio">portfolio</a></li>
			<li class="menu__items"><a href="#about">about</a></li>
			<li class="menu__items"><a href="#contact">contact</a></li>
		</ul>
	</div>
	<div class="container-title">
		<h2 class="header__h2">Welcome To Our Studio!</h2>
		<h1 class="header_h1">it's nice  to meet you</h1>
		<div class="header__button"><a href="#about">tell me more</a></div>
	</div>
</div>
<div class="service" id="service">
	<div class="container-service">
		<h2 class="service__h2">Services</h2>
		<h4 class="service__h4">Proin iaculis purus consequat sem cure. </h4>
		<div class="container-asides">
			<div class="service__aside-1">
				<div class="aside__icon-1"></div>
				<h4 class="aside__h4">E-Commerce</h4>
				<p class="aside__p">
					Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
				</p>
			</div>
			<div class="service__aside-2">
				<div class="aside__icon-2"></div>
				<h4 class="aside__h4">Responsive Web</h4>
				<p class="aside__p">
					Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
				</p>
			</div>
			<div class="service__aside-3">
				<div class="aside__icon-3"></div>
				<h4 class="aside__h4">Web Security</h4>
				<p class="aside__p">
					Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
				</p>
			</div>
		</div>
	</div>
</div>
<div class="portfolio" id="portfolio">
	<div class="container-portfolio">
		<h2 class="portfolio__h2">Our portfolio</h2>
		<h4 class="portfolio__h4">Proin iaculis purus consequat sem cure.</h4>
		<div class="container__works">
			<div href="#"class="work-1 works">
				<div class="work-1__foto work__fotos"></div>
				<div class="work__name">Ebony & Ivory</div>
				<div class="work__type">Branding</div>
			</div>
			<div href="#" class="work-2 works">
				<div class="work-2__foto work__fotos"></div>
				<div class="work__name">Smart Stationary</div>
				<div class="work__type">Print Design</div>
			</div>
			<div href="#" class="work-3 works">
				<div class="work-3__foto work__fotos"></div>
				<div class="work__name">Clever Poster</div>
				<div class="work__type">Print Design</div>
			</div>
			<div href="#" class="work-4 works">
				<div class="work-4__foto work__fotos"></div>
				<div class="work__name">Vinyl Record</div>
				<div class="work__type">Product Mock-Up</div>
			</div>
			<div href="#" class="work-5 works">
				<div class="work-5__foto work__fotos"></div>
				<div class="work__name">Treehouse Template</div>
				<div class="work__type">Web Design</div>
			</div>
			<div href="#" class="work-6 works">
				<div class="work-6__foto work__fotos"></div>
				<div class="work__name">Burned Logo</div>
				<div class="work__type">Branding</div>
			</div>
		</div>
	</div>
</div>
<div class="about-us" id="about">
	<div class="container-about-us">
		<h2 class="about__h2">About us</h2>
		<h4 class="about__h4">Proin iaculis purus consequat sem cure.</h4>
		<div class="container-history">
			<div class="event-1 events">
				<div class="events_title">
					<div class="events__date">July 2010</div>
					<h3 class="events__h3">Our Humble Beginnings</h3>
					<p class="events__text">
						Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.
					</p>
				</div>
				<div class="event-1__foto event__fotos"></div>
			</div>
			<div class="event-2 events">
				<div class="event-2__foto event__fotos"></div>
				<div class="events_title">
					<div class="events__date">January 2011</div>
					<h3 class="events__h3">Facing Startup Battles</h3>
					<p class="events__text">
						Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit aenean 
						rhoncus posuere odio in tincidunt.
					</p>
				</div>
			</div>
			<div class="event-3 events">
				<div class="events_title">
					<div class="events__date">December 2012</div>
					<h3 class="events__h3">Enter The Dark Days</h3>
					<p class="events__text">
						Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni.
					</p>
				</div>
				<div class="event-3__foto event__fotos"></div>
			</div>
			<div class="event-4 events">
				<div class="event-4__foto event__fotos"></div>
				<div class="events_title">
					<div class="events__date">february 2014</div>
					<h3 class="events__h3">Our Triumph</h3>
					<p class="events__text">
						Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit aenean.
					</p>
				</div>
			</div>
			<div class="container-story-continues events">
				<div class="story-continues">
					<div class="story-continues__text">
						Our <br>
						Story<br>
						Continues<br>
						...
					</div>
				</div>
			</div>
		</div>
	</div>
</div>
<div class="our-team">
	<div class="container-team">
		<h2 class="our-team__h2">our amazing team</h2>
		<h4 class="our-team__h4">Proin iaculis purus consequat sem cure.</h4>
		<div class="container-people">
			<div class="team__human-1 team__human">
				<div class="team-human__foto-1 team-human__foto"></div>
				<div class="name">Kimberly Thompson</div>
				<div class="profesion">Marketer</div>
				<div class="container-social-icon">
					<div class="twitter-icon social-icon"><a href=""></a></div>
					<div class="facebook-icon social-icon"><a href=""></a></div>
					<div class="pinterest-icon social-icon"><a href=""></a></div>
				</div>
			</div>
			<div class="team__human-2 team__human">
				<div class="team-human__foto-2 team-human__foto"></div>
				<div class="name">Rico Massimo</div>
				<div class="profesion">Coder</div>
				<div class="container-social-icon">
					<div class="twitter-icon social-icon"><a href=""></a></div>
					<div class="facebook-icon social-icon"><a href=""></a></div>
					<div class="google-plus-icon social-icon"><a href=""></a></div>
				</div>
			</div>
			<div class="team__human-3 team__human">
				<div class="team-human__foto-3 team-human__foto"></div>
				<div class="name">Uku Mason</div>
				<div class="profesion">Graphic Designer</div>
				<div class="container-social-icon">
					<div class="twitter-icon social-icon"><a href=""></a></div>
					<div class="facebook-icon social-icon"><a href=""></a></div>
					<div class="pinterest-icon social-icon"><a href=""></a></div>
				</div>
			</div>
			<div class="our-team__text">
				Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit
				aenean rhoncus posuere odio in tincidunt proin iaculis.
			</div>
		</div>
	</div>
</div>
<div class="logos">
	<div class="container-logos">
		<div class="logo-1 logo"><a href="https://envato.com"></a></div>
		<div class="logo-2 logo"><a href="https://wordpress.org/"></a></div>
		<div class="logo-3 logo"><a href="https://tutsplus.com"></a></div>
		<div class="logo-4 logo"><a href="http://microlancer.com"></a></div>
	</div>
</div>
<div class="contacts" id="contact">
	<div class="container-contacts">
		<h2 class="contacts__h2">Contact us</h2>
		<h4 class="contacts__h4">Proin iaculis purus consequat sem cure.</h4>
		<form action="#">
			<input type="text" class="name-inpt" placeholder="your name*">
			<input type="text" class="email-inpt" placeholder="your E-mail*">
			<input type="text" class="subject-inpt" placeholder="your subject*">
			<textarea type="text" class="message-inpt" placeholder="your message*"></textarea> 
			<button class="contacts__send"><a href="">Send Message</a></button>
		</form>
	</div>
</div>
<div class="footer">
	<div class="container-footer">
		<span class="footer__text">&copy Copyright 2014 FreebiesXpress.com</span>
		<div class="container-social-icon">
			<div class="twitter-icon social-icon"><a href=""></a></div>
			<div class="facebook-icon social-icon"><a href=""></a></div>
			<div class="pinterest-icon social-icon"><a href=""></a></div>
			<div class="google-plus-icon social-icon"><a href=""></a></div>
		</div>
	</div>
</div>

</body>
</html>